### DISCUSSION TITLE
How are multiple marriages and half-siblings displayed?

### DISCUSSION THREAD

**Discussion by LukeScania** [PeWu/topola-viewer#88] [Q&A]
We are currently evaluating Topola as a follow-up solution for a current genealogy tree which has been developed 9 years ago. I have a few questions about specific use-cases:
1. How are multiple marriages/divorces displayed?
2. How are half-siblings displayed?

In our current solution we have tabs for alternative spouses. half siblings are displayed next to each other without being connected.

**Comment by PeWu** [comment]
Here is an example:
https://pewu.github.io/topola-viewer/#/view?indi=Bernoulli-16&source=wikitree

Johannes Bernoulli had two wives. He had children in both marriages.
